Mathlib/Algebra/Field/Defs.lean

```diff
@@ -156,32 +183,5 @@ theorem smul_one_eq_coe (A : Type*) [DivisionRing A] (m : ℚ) : m • (1 : A) =
 
 end Rat
 
-end DivisionRing
-
-section OfScientific
-
 instance RatCast.toOfScientific [RatCast K] : OfScientific K where
   ofScientific (m : ℕ) (b : Bool) (d : ℕ) := Rat.ofScientific m b d
```

Comment:
Note for this PR: is it possible to have an `ofScientific` instance for `ℚ≥0` (or types with an `NNRatCast` instance)? I guess it would probably require interpreting `-1.2` as `1.2`.

Reply:
I think that would be a big trap. Certainly if people want it they can add it, but I won't do so myself